test/test-temp-directory.js

```diff
@@ -55,6 +56,8 @@ test('tempDirectory.create --tmpDir:', function (t) {
     fs.stat(ctx.path, function (err, stats) {
       t.error(err);
       t.ok(stats.isDirectory(), 'the path should exist and be a folder');
+      rmdir('./thisisatest', function () {
```

Comment:
We're excluding `test/.*` in .gitignore, so why not make the path `test/.thisisatest`.